### PR TITLE
add support for write events to AWS elasticsearch

### PR DESCRIPTION
**Pull request by m-richo**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -184,12 +208,22 @@ func mustEnv(name string) string {
...
 		App:     mustEnv("APP_NAME"),
 		Stream:  mustEnv("STREAM_NAME"),
 		Table:   mustEnv("TABLE_NAME"),
 		ConnStr: mustEnv("CONNECTION_STRING"),
 		ESURL:   mustEnv("ES_URL"),
+
+		ESregion:    os.Getenv("ES_AWS_REGION"),
+		ESaccesskey: os.Getenv("ES_AWS_ACCESS_KEY_ID"),
+		ESsecretkey: os.Getenv("ES_AWS_SECRET_ACCESS_KEY"),
+
```
Comment (aeijdenberg) [inline main.go:226]: These should be `mustEnv(...)` instead of `os.Getenv(...)` and then you can remove the first 4 lines of this function.

**Review by aeijdenberg — APPROVED** [review]
LGTM with minor change.

File: main.go (hunk 1 above)
Comment (m-richo) [reply main.go:226]: these 3 ENV vars are only used if you are writing to an AWS ES cluster.  When running locally they aren't needed so i thought it best not to use mustEnv...

Comment (aeijdenberg) [reply main.go:226]: Ah - I see those are slightly different names than the others above. LGTM

**Review by gordcorp — APPROVED** [review]
Looks fine to me. It'd be nice if it wasnt just reliant on AWS_* env vars, and also supported other aws auth methods such as AWS_PROFILE and credential files like we usually use locally, but I guess its not trivial so all good.

**Pull request merged**